Documentation/source/Error logging guide.md

```diff
@@ -0,0 +1,20 @@
+Error Logging
+=============================
+
+The Gini Capture SDK logs errors to the Gini Pay API when the default networking implementation is used (see the UI with Networking (Recommended) section in Integration guide). We log only non-sensitive information like response status codes, headers and error messsages.
+
+You can disable the default error logging by passing false to `GiniConfiguration().giniErrorLoggerIsOn`.
+
+If you would like to get informed of error logging events you need to set `GiniConfiguration().customGiniErrorLoggerDelegate` which confirms to `GiniCaptureErrorLoggerDelegate`:
+
+```swift
+customGiniErrorLoggerDelegate
```

Comment:
I think this is not needed here.

Reply:
Do you suggest keeping only the code example?